### PR TITLE
[FIX] Yahoo! Finance

### LINKED ISSUE #14: Yahoo Finance widget fails to download the data
Hi, Yahoo Finance widget fails to download the data no matter what the symbol or date you pick: " failed to download data (HTTP Error -1). Wrong ..."

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
Fixes #14 .

##### Description of changes
TODO:
- ~Remove first commit (`DataFrame` to `Table` function) (waiting https://github.com/biolab/orange3/pull/2836 to be merged).~
- ~Resolution (granularity): remove or fix~ removed

This widget works on only **Orange 3.9 or newer**.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kernc** [comment]
Granularity, if can't choose it, remove.

File: orangecontrib/timeseries/datasources.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
...
 from enum import Enum
 
+import pandas_datareader.data as web
+
 from Orange.data import Domain
 from orangecontrib.timeseries import Timeseries
+from orangecontrib.timeseries.pandas_compat import table_from_frame
 
 
 class DataGranularity(Enum):
```
Comment (kernc) [inline orangecontrib/timeseries/datasources.py]: If unused, remove.

Comment (jerneju) [reply orangecontrib/timeseries/datasources.py]: Yes, it is unused. And method `quandl_data` is unused as well.

File: orangecontrib/timeseries/datasources.py [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-class DataGranularity(Enum):
-    DAILY, \
-    WEEKLY, \
-    MONTHLY, \
-    DIVIDENDS_ONLY = 'dwmv'
+try:
+    from Orange.data.pandas_compat import table_from_frame
+except ImportError:
+    log.error("Yahoo Finance requires Orange >= 3.9")
```
Comment (kernc) [inline orangecontrib/timeseries/datasources.py]: Raise RuntimeError. Better not show the widget than to just crash later on.

**Comment by ajdapretnar** [comment]
Perhaps once this and Datasets PR are merged, you could think of bumping a version. :)

**Pull request merged**